src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsKernel.hpp

```diff
@@ -330,7 +360,6 @@ class SinglePhase :
     GEOSX_UNUSED_VAR( k );
     real64 maxForce = 0;
 
-//    CONSTITUTIVE_TYPE::KernelWrapper::DiscretizationOps::template fillLowerBTDB< numNodesPerElem >( stack.localJacobian );
     CONSTITUTIVE_TYPE::KernelWrapper::DiscretizationOps::template fillLowerBTDB< numNodesPerElem >( stack.localJacobian );
```

Comment:
We still need to remove the symmetry assumption, which breaks down for many of our plasticity models.

Reply:
Full `BTDT` added.